simpl/db/mongodb.py

```diff
@@ -115,6 +115,55 @@ def scrub(data):
         raise ValidationError("Input '%s' not permitted: %s" % (data, exc))
 
 
+def build_text_search(strings, name_field='name'):
```

Comment:
Just to be clear, I am basically just brainstorming here, not suggesting that these necessarily be implemented before we merge this.